src/lib/device_tree/src/lib.rs

```diff
@@ -16,7 +16,6 @@
 //! For more information about Device Tree Standard, see https://www.devicetree.org/.
 
 #![no_std]
-#![deny(warnings)]
```

Comment:
Is it possible to leave this in?

Reply:
Yes, reverted the change (I forgot to revert it after development)